spaces/S000156/properties/P000104.md

```diff
@@ -0,0 +1,19 @@
+---
+space: S000156
+property: P000104
+value: true
+---
+
+The topology on the space can be generated by the symmetric:
+
+$$d(m,n) = \begin{cases}
+0 & \text{if }x = y\\
+1 & \text{if }x \neq y\text{ and }x,y\in \omega\times\omega\\
+1 & \text{if }x \in \omega\times\omega\text{ and }y = \infty'\\
+1 & \text{if }y \in \omega\times\omega\text{ and }x = \infty'\\
+1 & \text{if }x = (\infty, m), y = (\infty, n),\text{ and }m \neq n\\
+\frac{1}{n+1} & \text{if }x = (n,m) \in \omega\times\omega\text{ and }y = (\infty, m)\\
+\frac{1}{n+1} & \text{if }y = (n,m) \in \omega\times\omega\text{ and }x = (\infty, m)\\
+\frac{1}{m+1} & \text{if }x = (\infty, m)\text{ and }y = \infty'\\
+\frac{1}{m+1} & \text{if }y = (\infty, m)\text{ and }x = \infty'\\
+\end{cases}$$
```

Comment:
(The other one looks more plausible, but I haven't thought about it.)

Reply:
@StevenClontz In that case I'll open an MSE post for this? It is sort of convoluted, but I couldn't think of a way to simplify the definition of the symmetric.

Reply:
Perhaps a few lines could be cut down by being sneaky and using a recursive `d(y,x) & \text{otherwise}`, so at least it doesn't *look* so bad? I think a question asking if these spaces are semimetrizable/symmetrizable on MSE would be reasonable though.

Reply:
@StevenClontz I now think it might be best to describe $d$ in English, something like "the distance from ... to ... is 1". It will likely be clearer what is actually happening that way. I'll just do an MSE question for S156. I think the semimetric for S15 is simple enough to justify writing the proof directly on pi-Base.